blog-posts/getting-started-gatsby-next-remix.md

```diff
@@ -0,0 +1,306 @@
+    //         {params: {path: 'second-blogpost'}},
+    //         ...
+    //         {params: {path: 'fifth-blogpost'}}
+    //     ],
+    const paths = getAllPostPaths()
+
+    return {
+        paths,
+        fallback: false
+    }
+
+export async function getStaticProps({params}: { params: PostsData }) {
+    const postData = await getPostData(params.path)
+    return {
+        props: {
+            postData
+        }
+    }
+}
+
+export default function Post({postData}: { postData: PostData }) {
+    return (
+            <div dangerouslySetInnerHTML={{__html: postData.contentHtml}}/>
+    )
+}
+```
+
+
+> 💡 The object returned by `getStaticProps` can include other keys besides `props`, such as `revalidate`. `revalidate` is set to `false` by default, but can also contain the amount in seconds after which a page re-generation can occur. In this way, individual static pages can be changed at a certain interval after being built. This is called [Incremental Static Regeneration (ISR)](https://vercel.com/docs/concepts/next.js/incremental-static-regeneration).
+
+
+#### **Remix**
+
+Remix supports only server side rendering. An example of this can be found in the section below.
+
+## Server Side Rendering (SSR)
+
+With server side rendering (SSR), the HTML is rebuilt for each request. All three frameworks support SSR.
+
+#### **Gatsby**
+
+Besides static site generation (SSG) you can also use server side rendering (SSR) for certain use cases in Gatsby. For this, the function `getServerData` must be built into a page, in which the data is requested from the server. This data can then be accessed in the actual page component with `serverData`. In the following the example from the [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/rendering-options/using-server-side-rendering/) is considered:
+
+```tsx
+import * as React from "react"
+
+const SSRPage = ({ serverData }) => (
+  <main>
+    <h1>SSR Page with Dogs</h1>
+    <img alt="Happy dog" src={serverData.message} />
+  </main>
+)
+
+export default SSRPage
+
+export async function getServerData() {
+  try {
+    const res = await fetch(`https://dog.ceo/api/breeds/image/random`)
+
+    if (!res.ok) {
+      throw new Error(`Response failed`)
+    }
+
+    return {
+      props: await res.json(),
+    }
+  } catch (error) {
+    return {
+      status: 500,
+      headers: {},
+      props: {}
+    }
+  }
+}
+```
+
+#### **Next.js**
+
+To use SSR in Next.js you have to use `getServerSideProps` instead of `getStaticProps`. This function must load the data and return it, too. The data can then be accessed in the page component (in the example via `data`). The following example shows a simple SSR implementation from the [Next.js documentation](https://nextjs.org/docs/basic-features/pages#server-side-rendering).
+
+```tsx
+function Page({ data }) {
+  // Render data...
+}
+
+// This gets called on every request
+export async function getServerSideProps() {
```

Comment:
I like the naming when you compare it with the SSR version `getStaticProps` 👌 That's something you could remind the reader about in the text too.